declarations/Facebook.json

```diff
@@ -15,12 +15,18 @@
       "executeClientScripts": true
     },
     "Privacy Policy": {
-      "fetch": "https://www.facebook.com/about/privacy",
+      "fetch": "https://mbasic.facebook.com/privacy/policy/printable/",
       "filter": [
-        "removeReturnToTopButtons",
-        "cleanUrls"
+        "removeEmptyAnchorsLinks",
```

Comment:
`document.querySelectorAll('[href="#"]')` length is 0, so i think this filter is not necessary.